### ISSUE TITLE
Groups / Profiles support

### ISSUE THREAD

**Issue by swiffer** [rickardandersson/rubric#1]
It would be greate to apply a group / multiple rules to a specific regex url (to set requests headers a+b for regex 1 and request header c for regex 2)

**Comment by rickardandersson** [comment]
Yeah, I agree that would be nice, but I purposefully skipped this feature when I implemented it due to it adding quite a lot of complexity in the UI. But if you have time and capacity, I'd welcome a PR to add this capability.

**Comment by rickardandersson** [comment]
@swiffer Better late than never, I suppose. Thanks to @albinhallden we now have profile support in release 1.2.0. I'll close this issue.
